.eleventy.js

```diff
@@ -1,53 +1,104 @@
+/*
+Copyright the Inclusive Learning Design Handbook copyright holders.
+
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-project/handbook.floeproject.org/raw/main/AUTHORS.md.
+
+Licensed under the New BSD license. You may not use this file except in compliance with this License.
+
+You may obtain a copy of the New BSD License at
+https://github.com/fluid-project/handbook.floeproject.org/raw/main/LICENSE.md.
+*/
+
 "use strict";
 
 const fs = require("fs");
-const fluidPlugin = require("@fluid-project/eleventy-plugin-fluid");
+
+const fluidPlugin = require("eleventy-plugin-fluid");
+const { EleventyRenderPlugin } = require("@11ty/eleventy");
+const MarkdownIt = require("markdown-it");
 const navigationPlugin = require("@11ty/eleventy-navigation");
+const rssPlugin = require("@11ty/eleventy-plugin-rss");
 const syntaxHighlightPlugin = require("@11ty/eleventy-plugin-syntaxhighlight");
+
+// Import transforms
+const htmlMinTransform = require("./src/transforms/html-min-transform.js");
 const parseTransform = require("./src/transforms/parse-transform.js");
 
+// Import data files
+const siteConfig = require("./src/_data/config.json");
+const getResourceLinks = require("./src/utils/getResourceLinks.js");
+
 module.exports = function (config) {
     config.setUseGitIgnore(false);
 
-    // Layouts
-    config.addLayoutAlias("default", "layouts/default.njk");
-
-    // Plugins
-    config.addPlugin(fluidPlugin);
-    config.addPlugin(navigationPlugin);
-    config.addPlugin(syntaxHighlightPlugin);
+    // Filters
+    const md = new MarkdownIt({
+        html: true,
+        quotes: "“”‘’"
+    });
+    config.addFilter("markdown", (content) => {
+        return md.render(content);
+    });
 
     // Transforms
+    config.addTransform("htmlmin", htmlMinTransform);
     config.addTransform("parse", parseTransform);
 
     // Passthrough copy
     config.addPassthroughCopy("src/_redirects");
-    config.addPassthroughCopy({"src/assets/images": "assets/images"});
+    config.addPassthroughCopy({"src/admin/config.yml": "admin/config.yml"});
```

Comment:
As we discussed before, content will be edited via Github pull requests. The admin page is unnecessary for this project and it's not working as well. Shall we remove the admin related code and documentation from this pull request?

Reply:
This seems like a reasonable step at this point. If anyone would like to add admin functionality into the site, they could find the necessary code in Trivet. I could also add a note to this effect in the Readme